### PR TITLE
ci: upgrade dompurify

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,37 @@
...
+        "url": "http://marsibarsi.me/"
+    },
+    "contributors": [
+        "Alex Inkin <alexander@inkin.ru>"
+    ],
+    "workspaces": [
+        "projects/*"
+    ],
     "scripts": {
+        "postinstall": "husky install && ngcc --async",
```
[the remark this answers is not in the document]
Comment (splincode) [reply package.json:28]: fixed

File: projects/ng-dompurify/tsconfig.lib.prod.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "./tsconfig.lib.json",
+    "compilerOptions": {
+        "declarationMap": false
+    },
+    "angularCompilerOptions": {
+        "enableIvy": false
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/ng-dompurify/tsconfig.lib.prod.json]: fixed

File: projects/ng-dompurify/tsconfig.lib.json [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
     "compilerOptions": {
         "outDir": "../../out-tsc/lib",
+        "declarationMap": true,
         "target": "es2015",
         "declaration": true,
         "inlineSources": true,
         "types": [],
         "lib": ["dom", "es2018"]
     },
     "angularCompilerOptions": {
```
Comment (nsbarsukov) [inline projects/ng-dompurify/tsconfig.lib.json:12]: https://angular.io/guide/angular-compiler-options#compilationmode

build

Comment (splincode) [reply projects/ng-dompurify/tsconfig.lib.json:8]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

**Pull request merged**